### PR TITLE
Add a Scope section to invariants

### LINKED ISSUE #3773: Normative Text in Invariants
I started a thread on the mailing list
https://mailarchive.ietf.org/arch/msg/quic/Zmu9uAyutk-dAz5wdAUjktTvfek/

And the general sense of it was that this text was insufficiently clear about the normative requirements it was creating. 

I believe this issue is editorial.

### PR DESCRIPTION
**Pull request by martinthomson**
I'm not committed to this.  I think that the document would be fine
without it, but it brings less of the manageability concerns into scope
than the proposed alternative.

Closes #3773.
Closes #3774.

### CODE REVIEW

**Comment by martinthomson** [comment]
That seems like a good way to do this.

File: draft-ietf-quic-invariants.md [hunk 1]
```diff
@@ -112,6 +112,23 @@ endpoints use QUIC packets to establish a QUIC connection, which is shared
 protocol state between those endpoints.
 
 
+# Scope
+
+To the extent that other entities are able to participate in communication
+between QUIC endpoints, a goal of the QUIC design is to provide technical
+measures that prevent interaction with a QUIC connection by entities other than
```
Comment (ianswett) [inline draft-ietf-quic-invariants.md:119]:
```suggestion
measures that limit interaction with a QUIC connection by entities other than
```

File: draft-ietf-quic-invariants.md [hunk 2]
```diff
@@ -112,6 +112,23 @@ endpoints use QUIC packets to establish a QUIC connection, which is shared
 protocol state between those endpoints.
 
 
+# Scope
+
+To the extent that other entities are able to participate in communication
+between QUIC endpoints, a goal of the QUIC design is to provide technical
```
Comment (ianswett) [inline draft-ietf-quic-invariants.md:118]:
```suggestion
A goal of the QUIC design is to provide technical
```

**Comment by janaiyengar** [comment]
I'd still like to see this trimmed down to just the second paragraph.

**Pull request closed without merging**